### PR TITLE
Added the ICU beds and historical estimations

### PR DESCRIPTION
**Pull request by jez4**

### CODE REVIEW

**Comment by gavento** [comment]
Could you, please, rename the files to be more descriptive? (hospital-capacity, estimated-infectious, column EstimatedInfectous)
For "source" files you want to check in, there is the sources directory.
Please update the Readme.

**Comment by gavento** [comment]
As I told, you, leave out: 25r0, 27b, 28, 28b (so no 28 should be there).
Also, 30, 31, and 04-02 are missing?
Please fix.

File: hospital-capacity.csv [hunk 1]
```diff
@@ -0,0 +1,60 @@
+Country,CriticalBedsPer100k,CapacityActiveInfectionPercent,CapacityNewInfectionsPerDayPer1000,Year,Source
```
Comment (gavento) [inline hospital-capacity.csv]: Please make this column "Code" too

Comment (jez4) [reply hospital-capacity.csv]: Done

**Comment by jez4** [comment]
I have added the codes to last file. Unfortunately I had it but copied incorrect file into data folder. Sorry for delaying.

**Review by hnykda — APPROVED** [review]
Looks good to me

**Comment by jez4** [comment]
@gavento I have the review from @hnykda. Should I merge it, or close the PR?

**Comment by mathijshenquet** [comment]
I think you should merge it

**Pull request merged**